node/resolvers/orderForm.ts

```diff
@@ -156,6 +156,42 @@ export const root = {
   },
 }
 
+export async function syncWithStoreLocale(
```

Comment:
Just to understand: this function is the same in `store-graphql`? If yes, could we leave a reference here? (maybe a link to the same part of the code)

Reply:
Sure, I'll add.